lib/ansible/modules/import_role.py

```diff
@@ -56,6 +56,14 @@
     type: bool
     default: yes
     version_added: '2.11'
+  public:
+    description:
+      - This option dictates whether the role's C(vars) and C(defaults) are exposed to the play.
+      - Variables are exposed to the play at playbook parsing time, and available to earlier roles and tasks as well unlike C(include_role).
+      - The default depends on the configuration option C(DEFAULT_PRIVATE_ROLE_VARS) configuration option.
```

Comment:
Does ref work here?
```suggestion
      - The default depends on the configuration option :ref:`default_private_role_vars`.
```
or just
```suggestion
      - The default depends on the configuration option C(DEFAULT_PRIVATE_ROLE_VARS).
```

Reply:
ref only works because we dont ship as collection and will always appear in the same doc site